starlight/templates/login_style.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Typically we will create a kind of `base.html` file that contains the HTML, HEAD, and BODY tags, then if we require a separate level of customization for a lower level of page in the hierarchy, we can create separate `*_base.html` pages. I would say to start with that and see where we go from there once we start working on other views! :+1: